src/libs/batch/batch-sdk.js

```diff
@@ -0,0 +1,17 @@
+export const getBatchSDK = () => {
```

Comment:
Pour info ce code ne fait pas que créer une balise script, il enregistre aussi batchSDK dans window.
Si on retire son code et qu'on ajoute un import de 'https://via.batch.com/v2/bootstrap.min.js' dans index.html, ça ne suffit pas (window.batchSDK is undefined).